src/pytorch_lightning/CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added support for saving sharded optimizer state dict outside of `DDPShardedStrategy` ([#14208](https://github.com/PyTorchLightning/pytorch-lightning/pull/14208))
 
 
+- Added an environment variable `PL_DISABLE_FORK` that can be used to disable all forking in the Trainer ([#14319](https://github.com/Lightning-AI/lightning/issues/14319))
```

Comment:
can you add it to the docs?

Reply:
oh! nvm.
there's an error

Reply:
I don't want to. This is just a workaround until https://github.com/pytorch/pytorch/issues/83973 lands. And we will open an issue in ray lightning to set this variable.

Reply:
@awaelchli I still would prefer it to be mentioned somewhere in the docs as even when the issue in torch lands, this would only be valid for the most recent torch version meaning that we may need to go with the current approach for a while (until all supported versions of torch are released past this issue).

Reply:
Yes, we are aware that this will have to stay for a while. But every user is a potential ray user and so documenting this environment variable in the "introduction" is not exactly great. It also can't exactly be documented in the ddp-fork section because the purpose is different. Then the only other solution is to document ALL environment variables in Lightning in one place. This will unnecessarily hold back this PR.

Reply:
Agreed. I think documenting ALL environment variables together with their purpose is something that would definitely be helpful but also definitely goes beyond this PR :)